os/drivers/lcd/st7701.c

```diff
@@ -21,5 +21,6 @@
 
 int check_lcd_vendor_send_init_cmd(struct mipi_lcd_dev_s *priv)
 {
+	set_power_inversion(priv, LCD_POWER_INVERSION_ON);
```

Comment:
This is cross reference, you should not apply this style.
you can refer to my comment, https://github.com/Samsung/TizenRT/pull/6910#discussion_r2238320226
then you can apply below style.
```
static int lcd_init(FAR struct lcd_dev_s *dev)
{
	FAR struct mipi_lcd_dev_s *priv = (FAR struct mipi_lcd_dev_s *)dev;
	struct mipi_dsi_device *dsi = dev->dsi_dev;
	priv->config->reset();
	bool inversion = dsi->get_inversion();
	set_power_inversion(priv, inversion);
	if (dsi->check_lcd_vendor_init(priv) == OK) {
		lcdvdbg("LCD Init sequence completed\n");
	} else {
		lcddbg("ERROR: LCD Init sequence failed\n");
	}
	priv->config->init();
	priv->config->lcd_enable();
	return OK;
}
```